module/VuFindSearch/src/VuFindSearch/Backend/EDS/Base.php

```diff
@@ -407,15 +409,16 @@ protected function call(
             $queryString,
             $headers,
             $message,
-            $messageFormat
+            $messageFormat,
+            $cacheable
         );
         return $this->process($response);
     }
 
     /**
      * Process EDSAPI response message
      *
-     * @param array $input The raw response from Summon
+     * @param string $input The raw response from Summon
```

Comment:
This comment mentions Summon -- might as well fix that too while we're in the area.

Reply:
Heh, managed to miss that somehow! Thanks for catching it, now tweaked.